### ISSUE TITLE
How to change controller setup?

### ISSUE THREAD

**Issue by dunx93** [ppeccin/WebMSX#70]
Hey there, I have an iPhone and using a razer kishi mfi controller. My problem is want to play metal gear 1 and 2 and need to use f1-f5 but my controller only sets up fast forward on both right triggers and slow motion on both left triggers. I just want to be able to adjust settings and want to know if possible?

**Comment by ppeccin** [comment]
Hello!

I'm afraid that is not possible in the current release.
Touch controls are the primary method for mobile touch devices, and you can
customize them to your needs.
The external controllers will work, but the problem is that the setup
screen in which you can customize them requires a mouse (pointer) and is
not available on touch-screen only devices. So for now there is no way to
change the default mappings directly on your device.

If you have a means of using a mouse on your device, you *could* try to
open the website in "desktop mode" then try to open the full settings
screen and change the mappings.
But I never tried that, it probably won't really work.
Sorry!

Paulo

**Comment by dunx93** [comment]
Are you able to make an update on this?

**Comment by ppeccin** [comment]
Yes, we could think of a way of improving this on a future release.
But I'm very busy at the moment, and also the next planned release will be
only some important bugfixes.

So that may take some time!

Paulo
